scripts/sync-workflow.js

```diff
@@ -58,3 +61,59 @@ function login(resolve, reject) {
     return resolve(resBody);
   });
 }
+
+function emptySync(loginResult) {
+  return function(resolve, reject) {
+    lr.actStart('EMPTY SYNC');
+
+    const myRequest = request.defaults({
+      headers: requestBodyUtils.getSyncRequestHeaders({
+        'X-FH-cuid': clientIdentifier,
+        'X-FH-sessionToken': loginResult.sessionToken
+      })
+    });
+
+    const reqBody = requestBodyUtils.getSyncRequestBody({
+      dataset_id: 'workorders',
+      query_params: {
+        "filter": {
+          "key": "assignee",
+          "value": "rkX1fdSH"
+        }
+      },
+      dataset_hash: '833c31caee5b87f4cbf9baeab0f5dce6814467ee',
+      meta_data: {
+        clientIdentifier: clientIdentifier
+      },
+      pending: [],
+      "__fh": {
+        "cuid": clientIdentifier,
+        "cuidMap": null,
+        "destination": `studio&fh_apptitle=WFM%20Demo%20Mobile%20App&url=${argv.app}`,
+        "sdk_version": "FH_JS_SDK/2.16.2-297",
+        "sessionToken": loginResult.sessionToken,
+        "appid": "zf47cmb7qhi36s3p4nws6q42",
+        "appkey": "b916e86a5366aff35dabab86654f4c7f41f82b0a",
+        "projectid": "zf47cmeatrf5iqvypq5tilq7",
+        "connectiontag": "0.0.1"
+      }
+    });
+
+    console.log(reqBody);
+    console.warn('About to post request');
+    myRequest.post({
+      url: `${argv.app}/cloud/workorders`,
+      body: reqBody,
+      json: true
+    }, (err, httpResponse, resBody) => {
+      console.warn('Got a reply');
+      if (err) {
+        console.error(err);
+        return reject(err);
+      }
+      console.warn(`#########################${util.inspect(resBody)}#########################`);
```

Comment:
`resBody` here is the string 'ok', but I'm expecting something like an object with an updates property.